### PR TITLE
Basket view

### PR DESCRIPTION
**Pull request by RadekLejba**

### CODE REVIEW

**Comment by RadekLejba** [comment]
Ran 93 tests in 2.156s

OK
Destroying test database for alias 'default'...
Name                                             Stmts   Miss  Cover
--------------------------------------------------------------------
baskets/__init__.py                                  0      0   100%
baskets/admin.py                                    32      1    97%
baskets/apps.py                                      4      0   100%
baskets/migrations/0001_initial.py                   8      0   100%
baskets/migrations/0002_auto_20170315_1136.py        7      0   100%
baskets/migrations/0003_basket_is_confirmed.py       5      0   100%
baskets/migrations/__init__.py                       0      0   100%
baskets/models.py                                   44      0   100%
baskets/templatetags/__init__.py                     0      0   100%
baskets/templatetags/filter_range.py                 4      0   100%
baskets/templatetags/get_item.py                     4      0   100%
baskets/tests/__init__.py                            0      0   100%
baskets/tests/test_admin.py                         39      0   100%
baskets/tests/test_apps.py                           6      0   100%
baskets/tests/test_models.py                        23      0   100%
baskets/tests/test_templatetags.py                   7      0   100%
baskets/tests/test_views.py                        187      0   100%
baskets/tests/utils.py                              25      0   100%
baskets/urls.py                                      3      0   100%
baskets/views.py                                   124      2    98%
command_runner.py                                   19     17    11%
menus/__init__.py                                    0      0   100%
menus/admin.py                                      23      0   100%
menus/migrations/0001_initial.py                     7      0   100%
menus/migrations/0002_auto_20170328_0934.py          6      0   100%
menus/migrations/__init__.py                         0      0   100%
menus/models.py                                     71      0   100%
menus/tests/__init__.py                              0      0   100%
menus/tests/test_admin.py                           14      0   100%
menus/tests/test_models.py                          41      0   100%
menus/tests/test_views.py                           15      0   100%
menus/tests/utils.py                                11      0   100%
menus/urls.py                                        3      0   100%
menus/views.py                                      13      0   100%
portal/__init__.py                                   0      0   100%
portal/admin.py                                      1      0   100%
portal/apps.py                                       4      0   100%
portal/migrations/0001_initial.py                    8      0   100%
portal/migrations/__init__.py                        0      0   100%
portal/models.py                                    18      0   100%
portal/pipeline.py                                  16      0   100%
portal/tests/test_apps.py                            6      0   100%
portal/tests/test_models.py                         12      0   100%
portal/tests/test_pipeline.py                       21      0   100%
portal/tests/test_views.py                          30      0   100%
portal/urls.py                                       4      0   100%
portal/views.py                                      4      0   100%
restaurants/__init__.py                              0      0   100%
restaurants/admin.py                                 9      0   100%
restaurants/apps.py                                  4      0   100%
restaurants/migrations/0001_initial.py               7      0   100%
restaurants/migrations/__init__.py                   0      0   100%
restaurants/models.py                               34      0   100%
restaurants/tests/test_apps.py                       6      0   100%
restaurants/tests/test_models.py                    37      0   100%
restaurants/tests/test_views.py                     30      0   100%
restaurants/urls.py                                  3      0   100%
restaurants/views.py                                 9      0   100%
service/__init__.py                                  0      0   100%
service/settings/__init__.py                         0      0   100%
service/settings/common.py                          32      0   100%
service/settings/development.py                      4      0   100%
service/urls.py                                      8      1    88%
--------------------------------------------------------------------
TOTAL                                             1052     21    98%

**Comment by RadekLejba** [comment]
`Ran 93 tests in 2.156s

OK
Destroying test database for alias 'default'...
Name                                             Stmts   Miss  Cover
--------------------------------------------------------------------
baskets/__init__.py                                  0      0   100%
baskets/admin.py                                    32      1    97%
baskets/apps.py                                      4      0   100%
baskets/migrations/0001_initial.py                   8      0   100%
baskets/migrations/0002_auto_20170315_1136.py        7      0   100%
baskets/migrations/0003_basket_is_confirmed.py       5      0   100%
baskets/migrations/__init__.py                       0      0   100%
baskets/models.py                                   44      0   100%
baskets/templatetags/__init__.py                     0      0   100%
baskets/templatetags/filter_range.py                 4      0   100%
baskets/templatetags/get_item.py                     4      0   100%
baskets/tests/__init__.py                            0      0   100%
baskets/tests/test_admin.py                         39      0   100%
baskets/tests/test_apps.py                           6      0   100%
baskets/tests/test_models.py                        23      0   100%
baskets/tests/test_templatetags.py                   7      0   100%
baskets/tests/test_views.py                        187      0   100%
baskets/tests/utils.py                              25      0   100%
baskets/urls.py                                      3      0   100%
baskets/views.py                                   124      2    98%
command_runner.py                                   19     17    11%
menus/__init__.py                                    0      0   100%
menus/admin.py                                      23      0   100%
menus/migrations/0001_initial.py                     7      0   100%
menus/migrations/0002_auto_20170328_0934.py          6      0   100%
menus/migrations/__init__.py                         0      0   100%
menus/models.py                                     71      0   100%
menus/tests/__init__.py                              0      0   100%
menus/tests/test_admin.py                           14      0   100%
menus/tests/test_models.py                          41      0   100%
menus/tests/test_views.py                           15      0   100%
menus/tests/utils.py                                11      0   100%
menus/urls.py                                        3      0   100%
menus/views.py                                      13      0   100%
portal/__init__.py                                   0      0   100%
portal/admin.py                                      1      0   100%
portal/apps.py                                       4      0   100%
portal/migrations/0001_initial.py                    8      0   100%
portal/migrations/__init__.py                        0      0   100%
portal/models.py                                    18      0   100%
portal/pipeline.py                                  16      0   100%
portal/tests/test_apps.py                            6      0   100%
portal/tests/test_models.py                         12      0   100%
portal/tests/test_pipeline.py                       21      0   100%
portal/tests/test_views.py                          30      0   100%
portal/urls.py                                       4      0   100%
portal/views.py                                      4      0   100%
restaurants/__init__.py                              0      0   100%
restaurants/admin.py                                 9      0   100%
restaurants/apps.py                                  4      0   100%
restaurants/migrations/0001_initial.py               7      0   100%
restaurants/migrations/__init__.py                   0      0   100%
restaurants/models.py                               34      0   100%
restaurants/tests/test_apps.py                       6      0   100%
restaurants/tests/test_models.py                    37      0   100%
restaurants/tests/test_views.py                     30      0   100%
restaurants/urls.py                                  3      0   100%
restaurants/views.py                                 9      0   100%
service/__init__.py                                  0      0   100%
service/settings/__init__.py                         0      0   100%
service/settings/common.py                          32      0   100%
service/settings/development.py                      4      0   100%
service/urls.py                                      8      1    88%
--------------------------------------------------------------------
TOTAL                                             1052     21    98%
`

**Comment by RadekLejba** [comment]
https://gist.github.com/RadekLejba/2f807a752b408ffaa93a2962886725fd

File: src/apof/baskets/templates/baskets/user_basket.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{% extends 'portal/base_body.html' %}
+{% load static %}
+
+{% block content %}
+    <table class="table table-hover table-condensed table-responsive">
+        {% for basket in object_list %}
+        {% if basket.is_confirmed %}
```
Comment (8r2y5) [inline src/apof/baskets/templates/baskets/user_basket.html]: Please fix indent.

File: src/apof/baskets/templates/baskets/user_basket.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                        </td>
+                        <td>
+                            {% include "baskets/_confirm_basket_button.html" %}
+                        </td>
+                    </tr>
+                </tfoot>
+                {% endif %}
+        {% endfor %}
+        {% empty %}
+        <h1>You have no orders yet.</h1>
```
Comment (8r2y5) [inline src/apof/baskets/templates/baskets/user_basket.html]: Please fix indent.

**Review by 8r2y5 — CHANGES_REQUESTED** [review]

**Comment by pwodzinski** [comment]
👍

**Pull request closed without merging**